### PR TITLE
Update retention-policies.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-o365seccomp/issues/751
Added three sentences for clarification.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/retention-policies.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ We're still working on retention in Teams, and additional features are coming. I
...
     
-In a Team, files that are shared in chat are stored in the OneDrive account of the user who shared the file. Files that are uploaded into channels are stored in the SharePoint site for the Team. Therefore, to retain or delete files in a Team, you need to create a retention policy that applies to the SharePoint and OneDrive locations. If you want to apply a policy to the files of just a specific team, you can choose the SharePoint site for the Team and the OneDrive accounts of users in the Team.
+In a Team, files that are shared in chat are stored in the OneDrive account of the user who shared the file. Files that are uploaded into channels are stored in the SharePoint site for the Team. Therefore, to retain or delete files in a Team, you need to create a retention policy that applies to the SharePoint and OneDrive locations. You would need to use a Retention Policy that targets SharePoint and OneDrive specifically. If you want to apply a policy to the files of just a specific team, you can choose the SharePoint site for the Team and the OneDrive accounts of users in the Team.
```
Comment (nenonix) [inline SecurityCompliance/retention-policies.md]:
```suggestion
In a Team, files that are shared in chat are stored in the OneDrive account of the user who shared the file. Files that are uploaded into channels are stored in the SharePoint site for the Team. Therefore, to retain or delete files in a Team, you need to create a retention policy that applies to the SharePoint and OneDrive locations specifically. If you want to apply a policy to the files of just a specific team, you can choose the SharePoint site for the Team and the OneDrive accounts of users in the Team.
```

Comment (JohanFreelancer9) [inline SecurityCompliance/retention-policies.md]:
```suggestion
In a team, files that are shared in chat are stored in the OneDrive account of the user who shared the file. Files that are uploaded into channels are stored in the SharePoint site for the team. Therefore, to retain or delete files in a team, you need to create a retention policy that applies to the SharePoint and OneDrive locations. You would need to use a Retention Policy that targets SharePoint and OneDrive specifically. If you want to apply a policy to the files of just a specific team, you can choose the SharePoint site for the team and the OneDrive accounts of users in the team.
```

**Comment by scanum** [comment]
@stephow-MSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**